lib/form_data.js

```diff
@@ -31,6 +31,7 @@ function FormData(options) {
   this._overheadLength = 0;
   this._valueLength = 0;
   this._valuesToMeasure = [];
+  this._entryList = {};
```

Comment:
We need to use a list rather than an array. The current spec allow you to append to the list multiple values for the same `key`